src/Tests/Moryx.Model.Tests/SqliteTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.Extensions.Logging;
+using Moryx.Model.InMemory;
+using Moryx.Model.Sqlite;
+using Moryx.Products.Model;
+using Moryx.Runtime.Endpoints.Databases.Endpoint.Services;
+using Moryx.Runtime.Kernel;
+using Moryx.TestTools.Test.Model;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moryx.Model.Tests
+{
+    [TestFixture]
+    public  class SqliteTests
+    {
+        private DbContextManager _dbContextManager;
+        private DatabaseConfigUpdateService _databaseConfigUpdateService;
+        private string _targetModelTypename;
+        private Type _configuratorType;
+
+        [SetUp]
+        public void Setup()
+        {
+
+            _dbContextManager = new DbContextManager(CreateConfigManager(), new LoggerFactory());
+            _databaseConfigUpdateService = new DatabaseConfigUpdateService(_dbContextManager);
+            _targetModelTypename = typeof(TestModelContext).FullName;
+            _configuratorType = typeof(SqliteModelConfigurator);
+        }
+
+        [Test]
+        public async Task SqliteCreateDatabaseShouldWork()
+        {
+            string databaseName = "TestCreateProductDatabase";
+            string datasource;
+            SqliteDatabaseConfig dbConfig;
+            SqliteModelConfigurator configurator;
+            InitializeSqliteTools(databaseName, out datasource, out dbConfig, out configurator);
+
+            var result = await configurator.TestConnection(dbConfig);
+            Assert.AreEqual(result, TestConnectionResult.ConnectionOkDbDoesNotExist);
```

Comment:
Here, expected and actual values are switched. This could lead to confusing outputs. Same probably goes for following calls of `AreEqual()`